.appveyor.sh

```diff
@@ -31,9 +31,10 @@ EOF
         ;;
 
     "test")
-        make test THREADS=$THREADS
         make binary-dist
-        7z a ghc-windows.zip *.tar.xz
+        curl https://ghc-artifacts.s3.amazonaws.com/tools/ghc-artifact-collector-x86_64-windows --output ghc-artifact-collector
```

Comment:
Is this tool tracked under version control somewhere? Is there any reason we can't just include it in the GHC repo?

Reply:
Agreed. I think we should eventually rebuild from source, for hermeticity and hence reproducibility. Shouldn't be a blocker though.

Reply:
The repo is here: https://github.com/tweag/ghc-artifact-collector. It has its own CI and commits on `master` are deployed (i.e. uploaded to S3) automatically. I think it may be handier to have this project in a separate repository because it's easier for me to hack on it and see changes almost instantly this way. If it's in GHC repo I'll be working through PRs and it's much much slower. We could include it in GHC repo when the helper stabilizes.

Reply:
The following two questions are orthogonal:

1. do we move it in-repo or keep it out?
2. do we use a prebuilt version pulled from somewhere?

The latter is independent of the former. If you push a prebuilt version somewhere and download it, you make it harder for others to fork (consider what happens if people want to try their own `ghc-artifcats-collector`). It's also not a hermetic setup.